internal/injector/aspect/join/declaration.go

```diff
@@ -49,7 +97,26 @@ func (i *valueDeclaration) AsCode() jen.Code {
 	return jen.Qual(pkgPath, "ValueDeclaration").Call(i.TypeName.AsCode())
 }
 
+var symbolNamePattern = regexp2.MustCompile(`\A(.+)\.([\p{L}_][\p{L}_\p{Nd}]*)\z`, regexp2.ECMAScript)
```

Comment:
Where was this taken from? If you created it yourself, can you add a regex101 link ?

Also: why start using `regex2` ?

Reply:
Go's `regex` does not support unicode categories >_>

Reply:
My bad, it actually supports those we need... :)